### DISCUSSION TITLE
why the bitrate is 64000

### DISCUSSION THREAD

**Discussion by PPQQXX** [onvif/testspecs#45] [Q&A] [answered]
Version:         ONVIF Device Test Tool version 22.12 rev. 28
Test Case ID:  MEDIA-2-1-9 
![image](https://github.com/onvif/testspecs/assets/32433430/7f129ad9-6b3d-49af-9faf-ef8c962e84f7)
the tools set the bitrate with 64000,and other params from device options,however, my device not support more than 20480.

**Comment by madhurao68** [comment, marked as answer]
Hi,
Device shall adapt to bitrate so the test is valid. BitrateLimit options appear later in a timeline than setting itself, so some Clients/Devices could ignore options.

Please find the Network Specification reference below.

![image](https://github.com/onvif/testspecs/assets/73811069/c81724ae-ce7a-4346-bea4-f3b6ff41eefb)
